microsoft/botframework-solutions
# [Virtual Assistant Template][C#] Fix Test QnA Maker button

## Description

In this PR will fix the `Test QnA Maker` button to send the correct message.

Fixes #1148 

## Testing Steps
1. Go to `AI/templates/Virtual-Assistant-Template/csharp`
2. Open the `Virtual Assistant solution`
3. Configurate all the pre-requisites to use the bot
4. The bot will display the intro card and the help buttons
5. Select the `Test QnA Maker` button
6. Check that the new message in the screen is `What is a Virtual Assistant?`
7. The bot will display successfully the request

## Checklist
**-**
